commcare_connect/opportunity/views.py

```diff
@@ -66,7 +66,10 @@ def get_success_url(self):
 
     def form_valid(self, form):
         form.instance.modified_by = self.request.user.email
-        return super().form_valid(form)
+        response = super().form_valid(form)
+        add_connect_users.delay(form.cleaned_data["users"], form.instance)
```

Comment:
Is there any reason for this task to be triggered when an Opportunity is edited? Is the UI for adding Phone Numbers a part of OpportunityEdit form?